src/app.rs

```diff
@@ -287,6 +359,27 @@ fn autoclick(
         }
     // Autoclick to emulate a humanlike clicks
     } else if app_mode == AppMode::Humanlike {
+        let click_x = click_coord.0;
+        let click_y: f64 = click_coord.1;
+        // move to target
+        #[cfg(debug_assertions)]
+        println!(
+            "Moving from {:?}/{:?} towards: {:?}/{:?}",
+            mouse_coord.0.to_f64(),
+            mouse_coord.1.to_f64(),
+            click_x,
+            click_y
+        );
+        move_to(
+            app_mode,
+            click_position,
+            (click_x, click_y),
+            is_moving_humanlike,
+            (mouse_coord.0.to_f64(), mouse_coord.1.to_f64()),
+            movement_delay_in_ms,
+        );
```

Comment:
This currently affects `AppMode::Humanlike` in `ClickPosition::Mouse`. Can it be moved to replace the 
```rust
                // move to final destination
                send(&EventType::MouseMove {
                    x: click_coord.0,
                    y: click_coord.1,
                });
```
found on Line 393 instead please? It will only affect `AppMode::Humanlike` `ClickPosition::Coord` in this way.

A check will also be needed to skip the `move_to()` if the cursor is already at the set click coordinates.